tests/saitests/py3/sai_qos_tests.py

```diff
@@ -3420,10 +3420,16 @@ def runTest(self):
                 send_packet(self, src_port_id, pkt, pkts_num_egr_mem +
                             pkts_num_leak_out + pkts_num_trig_egr_drp - 1 - margin)
             else:
+                if check_leackout_compensation_support(asic_type, hwsku):
+                    pkts_num_leak_out = 0
                # send packets short of triggering egress drop
                 send_packet(self, src_port_id, pkt, pkts_num_leak_out +
                             pkts_num_trig_egr_drp - 1 - margin)
-
+                if check_leackout_compensation_support(asic_type, hwsku):
+                    time.sleep(5)
```

Comment:
why does it need to sleep 5s here?

Reply:
inheritance from legacy code, sleeping for waiting for counter refresh